inc/savedsearch.class.php

```diff
@@ -595,33 +584,19 @@ function getParameters($ID) {
       return false;
    }
 
-   /**
-    * Mark saved search as default view for the current user
-    *
-    * @param integer $ID ID of the saved search
-    *
-    * @deprecated since version 9.2; @see SavedSearch::markDefault()
-    *
-    * @return void
-   **/
-   function mark_default($ID) {
```

Comment:
Deprecated methods should not be dropped in current version. But the ones in the current class are specific since the class itself has been renamed.

I guess keeping them may help to migrate anyways (and all methods deprecated in 9.1.2 should be removes as of 9.1..3)